### PR TITLE
Configure Zappr

### PR DESCRIPTION
**Pull request by elauria**
#3 This is a reduced example of a [configuration file](https://zappr.readthedocs.io/en/latest/setup) for Zappr.
If you have questions you can reach us

  * in our [Gitter channel](https://gitter.im/zalando/zappr)
  * on [Twitter](https://twitter.com/zalando_zappr)

These are the features we currently support:

## Approvals

  * Approve pull requests with comments and configurable pattern
  * "Veto" (ie. block) pull requests with comments and configurable pattern
  * Configure how many approvals your pull request needs before merging
  * Extensively configure who can approve or veto in your project

## Pull Request guidelines

  * Block a PR until it has the required set of labels (e.g. `reviewed`)
  * Block a PR as long as it has unfinished tasks

## Specification guidelines

  * Configure minimum length of pull request title and description
  * Configure if pull request description can be the same as pull request template

## Commit message guidelines

  * Configure patterns that all commits in a pull request must match

## Automatic branch creation

  * Automatically create a branch in your repository for every issue

### CODE REVIEW

**Comment by elauria** [comment]
👍

**Comment by elauria** [comment]
👍

File: .zappr.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#   by default, the pull request opener can approve its own pull request
+#   uncomment this line to ignore approvals from the pull request opener
+#   (there is also `last_committer` and `both`)
+#   ignore: pr_opener
+
+#   by default, zappr counts everyone as a valid approver
+#   uncomment these lines to change it to only collaborators
+#   (you can also specify `orgs` and `users` and define `groups`)
+#   from:
+#     collaborators: true
```
Comment (elgris) [inline .zappr.yaml]: Do you need extra configuration, like in other projects? (e.g. https://github.com/zalando-zmon/zmon-worker/blob/master/.zappr.yaml)

**Comment by elauria** [comment]
👍

File: .zappr.yaml (hunk 1 above)
Comment (elauria) [reply .zappr.yaml]: oh didnt even look at it, thought it had it already xD

**Comment by elgris** [comment]
👍

**Pull request closed without merging**